examples/blend/main.go

```diff
@@ -0,0 +1,193 @@
+// Copyright 2023 The Ebiten Authors
```

Comment:
Ebitengine

Reply:
Hm, I was referring to the other examples where the old name was kept.

Reply:
Fixed it here rn.

Reply:
> Should we update it everywhere in the examples?

You should not update the existing license comments.

Reply:
> You should not update the existing license comments.
I mean not here, in the separate issue.
Or there's a special reason why we can't never change them?

Reply:
> Or there's a special reason why we can't never change them?

I just follow other OSS's rules like Chromium: the copyright comment (year) is determined when the file is created, and is not updated basically.